scripts/sync_db.py

```diff
@@ -35,49 +62,28 @@ def run():
                 except Exception:
                     tools = ""
 
-                try:
-                    sha1sum = hashlib.sha1()
-                    with open(f"database/entries/{game}/{romfile}", 'rb') as source:
-                        block = source.read(2**16)
-                        while len(block) != 0:
-                            sha1sum.update(block)
-                            block = source.read(2**16)
-                        sha1 = sha1sum.hexdigest()
-                        print("SHA1:", sha1)
-                except Exception:
-                    sha1 = ""
+                _get_sha1_hash(game, romfile)
 
                 if "tags" not in data:
                     data["tags"] = []
+
                 if "platform" not in data:
                     print("no platform")
                     data["platform"] = "GB"
+
                 if "developer" not in data:
                     data["developer"] = ""
+
                 if "typetag" not in data:
                     data["typetag"] = "game"
+
                 if "title" not in data:
                     print("Warning: no title")
                     data["title"] = ""
 
-            try:
-                # Check if an entry already exists with the given slug
-                entry = Entry.objects.get(slug=data["slug"])
-                # And update its values
-                entry = Entry(
-                    slug=data["slug"],
-                    platform=data["platform"],
-                    developer=data["developer"],
-                    typetag=data["typetag"],
-                    title=data["title"],
-                    tags=data["tags"],
-                    basepath=folder,
-                    devtoolinfo=tools,
-                )
-                updated += 1
-            except Exception:
-                # otherwise, create a new entry
-                entry = Entry(
+            _, created = Entry.objects.update_or_create(
```

Comment:
Can we clarify here what is update_or_crate returning so it's more clear how we are using it?